### DISCUSSION TITLE
WebUI admin/user mode switching between

### DISCUSSION THREAD

**Discussion by whabby** [arendst/Tasmota#18055] [Q&A] [answered]
I changed WebUI with MQTT command to user mode. Is it possible to switch back to admin mode with http-command? I tried with te command http://<IP>/cm?cmnd=webserver%202 but I didn't had success. Tried the command http://<IP>/cm?cmnd=Power%20Toggle, this also was not working. Are http-commands not possible, if GUI of Tasmota (here V 10.1.0) is in user-mode?
Greetings

**Comment by barbudor** [comment, marked as answer]
Once switched to User-mode, HTTP commands are totally disabled.
Only MQTT or Serial command can allow to switch back
